examples/hello-triangle/main.rs

```diff
@@ -142,6 +142,13 @@ fn main() {
     #[cfg(not(target_arch = "wasm32"))]
     {
         env_logger::init();
+
+        #[cfg(feature = "subscriber")]
+        {
+            let chrome_tracing_dir = std::env::var("WGPU_CHROME_TRACING");
+            wgpu::util::initialize_default_subscriber(chrome_tracing_dir.ok());
```

Comment:
I don't think we need to show it off in hello- examples. Just framework is OK and most useful anyway.

Reply:
The reason I added this here too is because once we switch to `tracing` entirely, env_logger won't do anything, and I would imagine we want logging enabled. I can remove this for now and we can revisit it later if you want.